lib/tools/adb-emu-commands.js

```diff
@@ -5,42 +5,46 @@ import path from 'path';
 
 let emuMethods = {};
 
-emuMethods.isEmulatorConnected = async function (udid) {
-  let emulators = await this.getConnectedEmulators();
-  if (!emulators.length) {
-    return false;
+emuMethods.getEmulators = async function() {
```

Comment:
This are the results of running get connected emulators 30 times on my local laptop
```javascript
let times = [];
for(let i = 0;i<30;i++) {
      let t1 = (new Date()).getTime();
      await adb.getConnectedEmulators();
      let t2 = (new Date()).getTime();
      times.push([t1, t2, t2-t1]);
 }
```

```json
[ [ 1492484336620, 1492484336647, 27 ],
  [ 1492484336647, 1492484336663, 16 ],
  [ 1492484336664, 1492484336683, 19 ],
  [ 1492484336683, 1492484336697, 14 ],
  [ 1492484336697, 1492484336719, 22 ],
  [ 1492484336719, 1492484336739, 20 ],
  [ 1492484336739, 1492484336753, 14 ],
  [ 1492484336753, 1492484336770, 17 ],
  [ 1492484336770, 1492484336792, 22 ],
  [ 1492484336792, 1492484336805, 13 ],
  [ 1492484336805, 1492484336819, 14 ],
  [ 1492484336819, 1492484336836, 17 ],
  [ 1492484336836, 1492484336857, 21 ],
  [ 1492484336857, 1492484336871, 14 ],
  [ 1492484336871, 1492484336887, 16 ],
  [ 1492484336887, 1492484336904, 17 ],
  [ 1492484336904, 1492484336925, 21 ],
  [ 1492484336925, 1492484336939, 14 ],
  [ 1492484336939, 1492484336952, 13 ],
  [ 1492484336952, 1492484336975, 23 ],
  [ 1492484336975, 1492484336991, 16 ],
  [ 1492484336991, 1492484337004, 13 ],
  [ 1492484337004, 1492484337019, 15 ],
  [ 1492484337019, 1492484337047, 28 ],
  [ 1492484337047, 1492484337068, 21 ],
  [ 1492484337068, 1492484337087, 19 ],
  [ 1492484337087, 1492484337107, 20 ],
  [ 1492484337107, 1492484337128, 21 ],
  [ 1492484337128, 1492484337141, 13 ],
  [ 1492484337141, 1492484337155, 14 ] ]
```